spec/controllers/api/v1/updates_controller_spec.rb

```diff
@@ -4,16 +4,18 @@
   render_views
 
   let(:pattern) {
-    [
-     { update_type: "TariffSynchronizer::TaricUpdate",
-       state: String,
-       filename: String
-     }.ignore_extra_keys!,
-     { update_type: "TariffSynchronizer::TaricUpdate",
-       state: String,
-       filename: String
-     }.ignore_extra_keys!
-    ].ignore_extra_values!
+    { updates:
+      [
+       { update_type: "TariffSynchronizer::TaricUpdate",
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.